array/DNAm/config/config.example.txt

```diff
@@ -1,10 +1,13 @@
 ## Data filepaths
-export DATADIR=<data path>${PROJECT}
+export DATADIR=<data path>
```

Comment:
In recent past I've found that people get confused with <example here> as some think that the "<" and ">" characters should be kept. I see a lot of documentation use this instead:

```
path/to/data
```

which is more intuitive I guess.